spec/graphql/queries/set_query_spec.rb

```diff
@@ -0,0 +1,69 @@
+describe 'getCollection' do
+  before(:all) do
```

Comment:
```suggestion
  before(:each) do
```

Always use `each` when creating data, so each **example** is independent.
In the "getCollection loads collections by ID" example the count was wrong because you made 1 invisible in "returns collection with only publicly visible media entries".